docs/howto/writing-an-application-from-scratch/index.adoc

```diff
@@ -0,0 +1,99 @@
+
+[id='writing-an-application-from-scratch_{context}']
+= Creating an application from scratch
+
+Creating a simple WildFly Swarm{ndash}based application with a REST endpoint from scratch.
+
+[discrete]
+== Prerequisites
+
+* JDK 8 or newer installed
+* Maven 3.3.x or newer installed
+
+[discrete]
+== Procedure
+
+. Create a directory for the application:
++
+--
+[source,bash,options="nowrap"]
+----
+$ mkdir myApp
+$ cd myApp
+----
+
+We recommend you start tracking the directory contents with Git.
```

Comment:
Maybe we should show `git init` then? :-)

Reply:
But that might lead us to showing things like `git commit` etc. etc., so probably a good idea to not dive into it too much.

Reply:
Okay, I'll add that to Related information.